src/Core/Models/Api/Request/Organizations/OrganizationUpdateRequestModel.cs

```diff
@@ -1,9 +1,10 @@
-﻿using Bit.Core.Models.Table;
+﻿using Bit.Core.Models.Interfaces;
+using Bit.Core.Models.Table;
 using System.ComponentModel.DataAnnotations;
 
 namespace Bit.Core.Models.Api
 {
-    public class OrganizationUpdateRequestModel
+    public class OrganizationUpdateRequestModel: IPermissions
```

Comment:
Space before the colon, ` : `

Reply:
Not applicable with IPermissions getting the boot